src/plugins/source.js

```diff
@@ -19,5 +20,6 @@ export default async function () {
         resolve()
       }
     })
+    return pathToLog
```

Comment:
We return the promise. You can't return the result from promise via `return`, you can return some result only via resolve/reject promise methods. Remove `return pathToLog` and move it to `resolve(pathToLog)`